cardano_clusterlib/clusterlib.py

```diff
@@ -1692,6 +1692,46 @@ def get_kes_period(self) -> int:
         """Return last block KES period."""
         return int(self.get_slot_no() // self.slots_per_kes_period)
 
+    def get_kes_period_info(
+        self, issue_op_cert: FileType, operational_certificate_issue_counter: FileType
+    ) -> Dict:
+        """Check an operational certificate and certificate issue counter file.
```

Comment:
Suggestion based on command description from `cardano-cli query kes-period-info`:
"Get information about the current KES period and node's operational certificate."